ios/BillingPlugin.mm

```diff
@@ -170,12 +171,14 @@ - (void) paymentQueue:(SKPaymentQueue *)queue updatedTransactions:(NSArray *)tra
 		NSString *token = transaction.transactionIdentifier;
 
 		switch (transaction.transactionState) {
-			case SKPaymentTransactionStatePurchased:
+			case SKPaymentTransactionStatePurchased: {
```

Comment:
do we need braces here ?

Reply:
for objective c this is the right syntax.

else it will show syntax error on initializing a variable inside the case.

line 180

Reply:
okay, understood